From 049f7bd81cc96509efedbc1726e8c793d85b0229 Mon Sep 17 00:00:00 2001
From: JuanCRM <jcros84@gmail.com>
Date: Sun, 17 May 2020 20:38:09 +0200
Subject: [PATCH] Subida a Bitbucket

---
 src/Controller/InicioController.php | 1 -
 1 file changed, 1 deletion(-)

diff --git a/src/Controller/InicioController.php b/src/Controller/InicioController.php
index 4f9ad02..13ec121 100644
--- a/src/Controller/InicioController.php
+++ b/src/Controller/InicioController.php
@@ -13,7 +13,6 @@ class InicioController extends AbstractController
     public function index()
     {
         return $this->render('default/default.html.twig');
-
     }
 
     /**
-- 
2.24.1.windows.2

